consensus/misc/eip1559/eip1559_test.go

```diff
@@ -218,3 +220,54 @@ func TestCalcBaseFeeOptimismHolocene(t *testing.T) {
 		}
 	}
 }
+
+// TestCalcBaseFeeMinBaseFee assumes all blocks are jovian blocks that have the minimum base fee feature enabled.
```

Comment:
Oh and one last before I forget -- would it make sense to build a test to go across the activation? That is, no minimum on one block, and then a minimum set once activation hits?